amr-wind/wind_energy/actuator/wing/wing_ops.cpp

```diff
@@ -168,4 +168,56 @@ void write_netcdf(
 #endif
 }
 
+void refresh_wing_position(VecList& vpoints, VecList fpoints, const int npts)
+{
+    for (int ip = 0; ip < npts; ++ip) {
+        // Move velocity points to latest force points
+        vpoints[ip].x() = fpoints[ip].x();
+        vpoints[ip].y() = fpoints[ip].y();
+        vpoints[ip].z() = fpoints[ip].z();
+    }
+}
+
+void new_wing_position_velocity(
+    VecList& points,
+    vs::Vector& vtr,
+    const int npts,
+    amrex::Real tn,
+    amrex::Real tnp1,
+    const std::string motion,
+    amrex::Real period,
+    vs::Vector svec)
```

Comment:
feels like a bunch of these args can be `const` so the next developer knows what is in/out.